### PR TITLE
Document how to reuse code

### PR DESCRIPTION
**Pull request by rzane**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -381,6 +381,38 @@ def build_and_override_root_json(overrides = {})
...
+Now, you just need to extend it:
+
+```ruby
+operation :post do
+  # ...
+  response 200 do
+    # ...
+  end
+
+  extend SwaggerResponses::AuthenticationError
```
Comment (fotinakis) [inline README.md]: Can you put this at the top of the `operation` block, just for clarity?

**Comment by fotinakis** [comment]
This is great, thanks! Tiny comment above.

**Comment by rzane** [comment]
done!

**Comment by fotinakis** [comment]
Thanks!

**Pull request merged**